fedocal/__init__.py

```diff
@@ -864,9 +870,30 @@ def add_meeting(calendar_name):
             day=form.meeting_date.data.day))
     elif flask.request.method == 'GET':
         form = forms.AddMeetingForm(timezone=tzone, calendars=calendars)
+        mtg_date = flask.request.args.get('date', None)
+        if mtg_date:
+            form.meeting_date.data = parser.parse(mtg_date).date()
+            form.meeting_date_end.data = parser.parse(mtg_date).date()
+
+        mtg_time = flask.request.args.get('time', None)
+        if mtg_time:
+            form.meeting_time_start.data = mtg_time
+            hours, minutes = mtg_time.split(':')[:2]
+            try:
+                hours = int(hours) + 1
```

Comment:
Hm, what if a user calls this with a meeting time that they really did intend.  Then you bump their meeting time up by an hour.

Perhaps this default suggestion should be set in the javascript code that calls this endpoint.